I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 3, in script call
    call act1
  File "game/original_scripts/script-act1.rpy", line 38, in script
    $ name = pname.strip()
  File "game/original_scripts/script-act1.rpy", line 38, in <module>
    $ name = pname.strip()
NameError: name 'pname' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 3, in script call
    call act1
  File "game/original_scripts/script-act1.rpy", line 38, in script
    $ name = pname.strip()
  File "renpy/ast.py", line 922, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "renpy/python.py", line 2218, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/original_scripts/script-act1.rpy", line 38, in <module>
    $ name = pname.strip()
NameError: name 'pname' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.4.1439
Tweleve 1.0
Tue Jun 22 01:44:51 2021
